src/offer_template.rs

```diff
@@ -11,9 +12,14 @@ struct OfferTemplate {
     constraints: String,
 }
 pub(crate) fn gpu_detection<CONFIG: RuntimeConfig>(config: &CONFIG) -> anyhow::Result<Option<Gpu>> {
-    let gpu_detection = gpu_detection::GpuDetection::init()?;
-    let gpu = gpu_detection.detect(config.gpu_uuid())?;
-    Ok(Some(gpu))
+    match gpu_detection::GpuDetection::init() {
+        Ok(gpu_detection) => {
+            let gpu = gpu_detection.detect(config.gpu_uuid())?;
+            Ok(Some(gpu))
+        }
+        Err(GpuDetectionError::LibloadingError(_)) => Ok(None),
```

Comment:
This `Ok(None)` will make `ya-runtime-ai test` pass
https://github.com/golemfactory/ya-runtime-ai/blob/85be431728c0a12135f3710984cc5a3efebba56a/src/main.rs#L202